modules/CodeTablesAdmin/src/client/AbstractCodeTablesAdminBrowser.coffee

```diff
@@ -174,13 +199,15 @@ class AbstractCodeTablesAdminBrowserController extends Backbone.View
 		@searchController = new CodeTablesAdminSimpleSearchController
 			model: new CodeTablesAdminSearch()
 			el: @$('.bv_codeTablesAdminSearchController')
-			urlRoot: "/api/codeTablesAdmin/#{@codeType}/#{@codeKind}"
+			urlRoot: @urlRoot
 			toDisplay: @toDisplay
 		@searchController.render()
 		@searchController.on "searchRequested", @handleSearchRequested.bind(@)
 		@searchController.on "searchReturned", @setupCodeTablesAdminSummaryTable.bind(@)
 		@searchController.on "createNewCodeTablesAdmin", @handleCreateNewCodeTablesAdminClicked.bind(@)
-	#@searchController.on "resetSearch", @destroyCodeTablesAdminSummaryTable
+		# If search on load, trigger a search for "*"
+		if @searchOnLoad
+			@searchController.doSearch('*')
```

Comment:
1. I think we should call `handleSearchRequested()` and then refactor handleSearchRequested a little bit to handle this new case.  The reason I say this is because `handleSearchRequested` is called programmatically in `handleBackToCodeTablesAdminBrowserClicked` and `handleConfirmDeleteCodeTablesAdminClicked` which could also benefit from this new behavior.

Reply:
Unrelated to this PR but in scope, but we should also hide the `bv_codeTablesAdminControllerContainer` when doing `handleSearchRequested`  which we aren't doing right now:

i.e. add this to `handleSearchRequested`

```
@$(".bv_codeTablesAdminControllerContainer").addClass "hide"
```

Reply:
Wondering if we could/should also add a "searchOnLoadDefaultSearchTerm" here.  i.e. I can imagine a code table search where the subclass wants to search for something other than "*" by default.  i.e. search by username or date? Just a thought and something to possibly hook into later even if it isn't perfect.

Reply:
Thanks - this cleared up that bug.
I took all of your suggestions and added a new "defaultSearchTerm" that simply pre-populates the search box, visibly to the user.

I tested changing `defaultSearchTerm` and having it with & without `searchOnLoad` and all the combinations work.